editor/project_manager.cpp

```diff
@@ -765,6 +765,7 @@ class ProjectDialog : public ConfirmationDialog {
 
 				set_title(TTR("Install Project:") + " " + zip_title);
 				get_ok()->set_text(TTR("Install & Edit"));
+				project_name->set_text(TTR(zip_title));
```

Comment:
`TTR()` means "Tools TRanslate", it's for translatable strings. It shouldn't be used here for `zip_title`, which can't be translated.

Reply:
thank you for the tip!